generated/rflx-arrays-generic_inner_message.ads

```diff
@@ -4,11 +4,11 @@ with RFLX.RFLX_Generic_Types;
 generic
    with package Types is new RFLX.RFLX_Generic_Types (<>);
 package RFLX.Arrays.Generic_Inner_Message with
-  SPARK_Mode
+  SPARK_Mode,
```

Comment:
Shouldn't we indent package aspects also with 3 spaces?

Reply:
Aspects are always indented by two spaces. That should be according to the [Ada style guide](https://en.m.wikibooks.org/wiki/Ada_Style_Guide/Source_Code_Presentation).

> Use two spaces as the basic unit of indentation for continuation lines.

Actually, our style guide is not consistent in that matter.